vector_data/polygon/CONTENTS.md

```diff
@@ -2,8 +2,8 @@
 
 Here are descriptions of the polygon vector data directories found in this subfolder.
 
-## `ak_protected_areas`
-Boundaries for Alaska protected areas (Parks, Refuges, Reserves, etc.) created by the Commission for Environmental Cooperation (CEC) and published in 2010 as part of larger pan-North America dataset. A protected area is an “area of land and/or sea especially dedicated to the protection and maintenance of biological diversity, and of natural and associated cultural resources, and managed through legal or other effective means. 
+## `protected_areas`
+Boundaries for protected areas in Alaska and western Canada (Parks, Refuges, Reserves, etc.) created by the Commission for Environmental Cooperation (CEC) and published in 2010 as part of larger pan-North America dataset. A protected area is an “area of land and/or sea especially dedicated to the protection and maintenance of biological diversity, and of natural and associated cultural resources, and managed through legal or other effective means. Organized by state or province.
```

Comment:
Last sentence should be "Organized by state, territory, or province."

Reply:
Good catch - thanks Mike!